server/src/controllers/courseInstance.ts

```diff
@@ -116,7 +140,7 @@ export async function getCourseInstance(req: Request, res: Response): Promise<vo
 }
 
 export interface CourseInstanceWithTeachers extends CourseInstance {
-  teacher: User
+  Users: Array<User>
```

Comment:
Could we consider using `{ name: string }` instead of the full `User` type here? Below, the `findAll` call only extracts the `name` attribute from the database, so this is unsound in terms of the TypeScript type system.